.mailmap

```diff
@@ -0,0 +1,55 @@
+# See https://github.com/git/git/blob/master/Documentation/mailmap.txt
+adil <adil@stockopedia.co.uk>
+adil <adil@stockopedia.co.uk> <adil@adil-ThinkCentre-A85>
+aG0aep6G <aG0aep6G@users.noreply.github.com>
+Alex Rønne Petersen <alex@alexrp.com> <alex@lycus.org>
+amaury <deadalnix@gmail.com> <amaury@fb.com>
+Anders Ronnbrant <anders.ronnbrant@gmail.com>
+Andrej Mitrovic <andrej.mitrovich@gmail.com>
+Andrey Zherikov <andrey.zherikov@gmail.com>
+biotronic <simen.kjaras@gmail.com> <biotronic@biotronic.(none)>
+Brian Schott <briancschott@gmail.com>
+Christian Juelg <c.juelg@gmail.com>
+Clement Courbet <legrosbuffle@somebod.com> <clem@somebod.com>
+Daniel Murphy <yebblies@gmail.com>
+Dave Gallets <davegallets@gmail.com>
+David <admin@dav1d.de> <admin@dav1d.de>
+Denis Shelomovskij <verylonglogin.reg@gmail.com>
+Ed McCardell <edmccard@hotmail.com>
+Hara Kenji <k.hara.pg+dev@gmail.com> <k.hara.pg@gmail.com>
+Iain Buclaw <ibuclaw@gdcproject.org> <ibuclaw@HP-G62.(none)>
+Iain Buclaw <ibuclaw@gdcproject.org> ibuclaw <ibuclaw@ubuntu.com>
+Ilya Yaroshenko <ilyayaroshenko@gmail.com>
+Jacob Carlborg <doob@me.com> <jacob.carlborg@gmail.com>
+Jakob Øvrum <jakobovrum@gmail.com>
+Jakob Øvrum <jakobovrum@gmail.com> <JakobOvrum@users.noreply.github.com>
+John Colvin <john.loughran.colvin@gmail.com>
+Jonathan M Davis <jmdavisProg@gmx.com> <jonathan.davis@pelco.com>
+Jordi Sayol <g.sayol@yahoo.es>
+Kai Nacke <kai@redstar.de>
+kinke <noone@nowhere.com>
+Mark Barbone <30397247+mb64@users.noreply.github.com>
+Mark Barbone <30397247+mb64@users.noreply.github.com> Mark Barbone <>
+Martin Nowak <code@dawg.eu> <dawg@dawgfoto.de>
+Mathias Lang <mathias.lang@sociomantic.com> <geod24@gmail.com>
+Mathias Lang <mathias.lang@sociomantic.com> <mathias.lang@epitech.eu>
+Mathias Lang <mathias.lang@sociomantic.com> <pro.mathias.lang@gmail.com>
+monarchdodra <monarchdodra@gmail.com> <monarchdodra@gmail.com>
+Petar Kirov <petar.p.kirov@gmail.com> <petar.p.kirov@gmail.com>
+ponce <aliloko@gmail.com> <aliloko@gmail.com>
+Rainer Schuetze <r.sagitario@gmx.de> <r.sagitario@gmx.de>
+Razvan Nitu <razvan.nitu1305@gmail.com> <razvan.nitu1305@gmail.com>
+Robert burner Schadek <robert.schadek@informatik.uni-oldenburg.de> <rburners@gmail.com>
+Robert burner Schadek <robert.schadek@informatik.uni-oldenburg.de> <robert.schadek@informatik.uni-oldenburg.de>
+Safety0ff <safety0ff.dev@gmail.com>
+Safety0ff <safety0ff.dev@gmail.com> <safety0ff@users.sourceforge.net>
+shoo <zan77137@nifty.com> <zan77137@nifty.com>
+Stefan Koch <Uplink.Coder@googlemail.com>
+Stefan Rohe <think@hotmail.de>
+Timothee Cour <timothee.cour@gmail.com> <timothee.cour2@gmail.com>
+Torarin <torarin@gmail.com>
+Ulrich Küttler <kuettler@gmail.com>
+Walter Waldron <WalterWaldron@users.noreply.github.com>
+Yazan Dabain <yazan.dabain@arabiaweather.com> <yazan.dabain@gmail.com>
+Михаил Страшун <public@dicebot.lv> <m.strashun@gmail.com>
+Yao Gómez <yao.gomez@gmail.com> <yaoltzin@gmail.com>
```

Comment:
Should we remove the dlang repo specific mailmaps then?

Reply:
https://github.com/dlang/dmd/pull/7452